app/src/webkit/java/org/mozilla/focus/webkit/matcher/BlocklistProcessor.java

```diff
@@ -76,8 +152,9 @@ private void extractSite(final JsonReader reader) throws IOException {
                     reader.beginArray();
 
                     while (reader.hasNext()) {
+                        // TODO: iOS blocklist parsing removes www. prefix
```

Comment:
Yup - #95, which seems to be unnecessary AFAICS. (And I forgot to remove this comment)